README.md

```diff
@@ -1,2 +1,16 @@
 # package-metadata-interoperability-collab-space
+
 The goal of the Package Metadata Interoperability Collab Space is to improve how JavaScript developers define their packages across the ecosystem. The group is currently working to better understand package.json and how developers use it to define their projects.
+
+## Current Initiatives
+
+1. [package.json research](https://github.com/openjs-foundation/package-json-research)
+   1. This initiative's goal is to research existing usage of `package.json` throughout the JavaScript ecosystem
+   2. With this research, the group can develop tooling, documentation, specifications, and much more
+   3. The linked repository contains all information for how to get involved and contribute
+2. [`@pkgjs/create-pkg` tooling](https://github.com/pkgjs/create-pkg)
```

Comment:
GitHub link is 404. is this a private repo?

Reply:
it's public, and https://github.com/pkgjs/create-pkg loads for me.